### PR TITLE
Move new styles (2)

### PR DESCRIPTION
**Pull request by svbergerem**
Moves all remaining styles from `app/assets/stylesheets/new_styles/` to  `app/assets/stylesheets/`. Again, remaining selectors with underscores are out of scope of this PR.

I will add comments to lines where I changed more than just the code style.

### CODE REVIEW

File: app/assets/stylesheets/interactions.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      line-height: $line-height-base;
+      vertical-align: middle;
+    }
+
+    [class^="entypo-"]:hover,
+    [class*="entypo-"]:hover {
+      color: $text;
+    }
+
+    &.hide_conversation i { font-size: $line-height-computed * 1.5; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `hide_conversation` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    &.delete_conversation i { font-size: $font-size-base * 1.5; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `delete_conversation` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    &.destroy_participation i { color: $black; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `destroy_participation` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    &.destroy_participation i:hover { color: $text-dark-grey; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `destroy_participation` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  }
+}
+
+.stream_container,
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `stream_container` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+.single-post-interactions {
+  .control-icons {
+    float: right;
+    z-index: 6;
+
+    .block_user,
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `block_user` should be written in lowercase with hyphens

Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `comment_report` should be written in lowercase with hyphens

Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `create_participation` should be written in lowercase with hyphens

Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `destroy_participation` should be written in lowercase with hyphens

Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `post_report` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    .destroy_participation,
+    .post_report {
+      display: inline-block;
+    }
+
+    > a:hover { text-decoration: none; }
+  }
+}
+
+.stream_element,
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `stream_element` should be written in lowercase with hyphens

File: app/assets/stylesheets/interactions.scss [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+.stream_element:hover .comment {
+  .control-icons {
+    @include transition(opacity);
+    opacity: 0;
+  }
+
+  &:hover .control-icons { opacity: 1; }
+}
+
+.stream_element,
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/interactions.scss]: Selector `stream_element` should be written in lowercase with hyphens

File: app/assets/stylesheets/new_styles/_forms.scss [hunk 9]
```diff
@@ -1,90 +0,0 @@
...
-    box-shadow: none;
-    color : $text-dark-grey;
-  }
-}
-
-/* autocomplete colors */
-input:-webkit-autofill{
-  background-color: #fff !important;
-  background-image: none !important;
-}
```
Comment (svbergerem) [inline app/assets/stylesheets/new_styles/_forms.scss]: In Chromium the background was still yellow so I removed these lines and I don't think that we have to override the background.

File: app/assets/stylesheets/new_styles/_interactions.scss [hunk 10]
```diff
@@ -1,59 +0,0 @@
...
-  a {
-    &:hover { text-decoration: none; }
-
-    [class^="entypo-"], [class*="entypo-"] {
-      color: $text-grey;
-      font-size: $font-size-base;
-      line-height: $line-height-base;
-      vertical-align: middle;
-      &:hover { color: $text; }
-      &.cross { font-size: $line-height-base; }
```
Comment (svbergerem) [inline app/assets/stylesheets/new_styles/_interactions.scss]: This element didn't exist anymore.

File: app/assets/stylesheets/interactions.scss [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+.comment,
+.photo {
+  .control-icons > a {
+    @include transition(opacity);
+    opacity: .8;
+  }
+
+  .control-icons > a:hover { opacity: 1; }
+}
```
Comment (svbergerem) [inline app/assets/stylesheets/interactions.scss]: Before we changed the opacity of the icons like this:
default: 0
stream element hovered: 0.8
icon hovered: 1

Now the opacity of the whole control-icons div changes:
default: 0
stream element hovered: 1

And the opacity of the icons is:
default: 0.8
icon hovered: 1

From the users POV this should be exactly the same.

File: app/assets/stylesheets/new_styles/_settings.scss [hunk 12]
```diff
@@ -1,31 +0,0 @@
-/* Specific styles for the settings pages (profile, user account, privacy, services) */
-#inner_account_delete {
-	width: 700px;
-}
```
Comment (svbergerem) [inline app/assets/stylesheets/new_styles/_settings.scss]: This element doesn't exist anymore. Before it was inside of the close account facebox.

File: app/assets/stylesheets/new_styles/_settings.scss [hunk 13]
```diff
@@ -1,31 +0,0 @@
...
-
-.as-selections #tags {
-	box-shadow: none;
-}
```
Comment (svbergerem) [inline app/assets/stylesheets/new_styles/_settings.scss]: The input field has no box-shadow anymore, so I removed these lines.

**Comment by jhass** [comment]
<3 Thanks

**Pull request merged**